### DISCUSSION TITLE
你们新开发的能自选服务器的版本支持win7吗？

### DISCUSSION THREAD

**Discussion by gl1031** [getlantern/lantern-forum-cn#519] [General]
如题！电脑里一款老财务软件公司必备！必须要用WIN7才能用
你们有没有考虑新开的自选服务器版本的lantern做一个WIN7版？
不然只能用老版lantern，还是不能自选不是？

**Comment by MaxMKMax** [comment]
@gl1031 您好. 这个问题呢. 暂时回答您应该是不支持的. 因为代码语言的不同所以无法同步更新window 7 的版本了.
但是我也希望可以支持. 我会尽量帮window7 的用户争取新版本的更新. 让我和技术团队商量一下. 
希望您能理解. 谢谢您一直都支持蓝灯. 谢谢您!
